pyproject.toml

```
@@ -32,6 +32,7 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
+fido2 = ">=1.1.2, <3"
```

```suggestion
fido2 = ">=1.1.2, <2"
```

For the SDK, I’ve reviewed the breaking changes and added tests for both versions to the CI. If you just want to fix #360, we should stick with v1. Otherwise we can just update to v2. Probably, the breaking changes don’t affect nitrokey-app2 either. I don’t think there’s merit in supporting both versions for nitrokey-app2 as an application.

---

Yes, but I would say that was wrong. The code in nitrokey-app2 was developed and tested against fido2 v1. Before nitrokey-sdk-py v0.3.2, the implicit fido2 requirement was `<2`. When I updated to nitrokey-sdk-py v0.3.2 in https://github.com/Nitrokey/nitrokey-app2/pull/341, I wasn’t aware that we don’t have a `fido2` bound in nitrokey-app2. Otherwise I would have added it (with `<2`).

---

AFAIS the breaking changes in fido2 v2 don’t affect nitrokey-app2 either: https://github.com/Nitrokey/nitrokey-app2/pull/362